recipe/meta.yaml

```diff
@@ -1,45 +1,51 @@
-{% set name = "<PLEASE ADD PKG NAME>" %}
-{% set version = "<PLEASE ADD PKG VERSION>" %}
+{% set name = "streamlit-mermaid" %}
+{% set version = "0.2.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: <sha256>
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/streamlit-mermaid-{{ version }}.tar.gz
```

Comment:
FYI, a PyPI project leads to the wrong home URL https://github.com/AI-Yash/st-chat (but was fixed here https://github.com/neka-nat/streamlit-mermaid/commit/6eac163199b384bc249da06c417459a74f00a2d6).
Another question is that the upstream source on github has additional folders in `streamlit_mermaid/frontend` but the pypi tarball has only `streamlit_mermaid/frontend/build`.
I wonder if the package isn't vulnerable somehow but I have no idea how to verify it.

UPD. Here is why only `streamlit_mermaid/frontend/build` is packaged https://github.com/neka-nat/streamlit-mermaid/commit/324873e3a8602fc108b72ff1a6a61a010cd7ad72